src/fipp/engine.cljc

```diff
@@ -5,6 +5,12 @@
             [fipp.deque :as deque]))
 
 
+(defn ^{:private true} unchunk [s]
```

Comment:
I'd suggest the `^:private` shorthand notation, but in general _everything_ in this file is private except `pprint-document`. I'd rather just document that fact in the readme than give the illusion that the other stuff is also public by hinting on one function as private.